### ISSUE TITLE
Missing a step somewhere - Gmail Workspace Account Authentication

### ISSUE THREAD

**Issue by pfeiff77** [simonrob/email-oauth2-proxy#231]
I have tried on Windows, Ubuntu and via python and I still get the same errors and I am sure I am just missing something really simple. I am not getting any prompts to authorize the oAuth either via cli or on the app.

In the logs I am getting the following but trying to track it down is not giving me any leads.

Doing SMTP to Google using the port 2465 that is in the config file. I am using my workspace account which is of my own domain but it is going through Google of course.

Here are the log entries:

<-- b'530-5.7.0 Authentication Required. For more information, go to\r\n'
<-- b'530-5.7.0 Authentication Required. For more information, go to\r\n'
<-- b'530-5.7.0 https://support.google.com/mail/?p=WantAuthError kd3-20020a05622a268300b0042e09c6b82dsm2181889qtb.24 - gsmtp\r\n'

Doing some searching is not coming up with anything. I do have the project in my workspace for the API and have the client_id and client_secret

I don't have an @gmail entry in my config file but something like the example below. I've also tried the advance google entry as well and no joy.

[myuser@mydomain]

Hopefully it is something simple that i am missing.

Thanks.

**Comment by simonrob** [comment]
Are you using a service account or a normal Gmail setup? If you're struggling to get set up it's always worth starting with the simplest method and getting that working first.

**Comment by pfeiff77** [comment]
Thanks for getting back to me. I am using just the standard gmail setup using my workspace account. No service account. I created the project in google and even did web and desktop app ones and copied the ids over but no joy.

For the config entry under the account entry I have tried using my email address and also just the @domain part for the info in the brackets.

Does the entry need to be @gmail.com? My domain of course isn't a @gmail.com.

I've tried using telnet to do a test to the smtp server and all the commands but the proxy just doesn't try to authenticate. I could try using another of my normal @gmail.com accounts but I do think it would still give me the same results. Nothing in my firewall is blocking the attempts to google or anything along those lines.
